fred2/sexp_tree.cpp

```diff
@@ -4915,11 +4923,14 @@ void sexp_tree::update_help(HTREEITEM h)
 	}
 
 	code = get_operator_const(tree_nodes[i].text);
-	str = help(code);
-	if (!str)
-		str = "No help available";
+	auto str = help(code);
+	if (!str) {
+		text.Format("No help available\r\n\r\n%s", nodeComment.c_str());
+	} else {
+		text.Format("%s\r\n\r\n%s", str, nodeComment.c_str());
```

Comment:
Same comments with the two blank lines.  If `nodeComment` will always be preceded by two blank lines, then they could be added up on line 4794.